<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmLightButton</class>
 <widget class="QWidget" name="frmLightButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="LightButton" name="widget1" native="true"/>
   </item>
   <item>
    <widget class="LightButton" name="widget2" native="true">
     <property name="bgColor" stdset="0">
      <color>
       <red>255</red>
       <green>107</green>
       <blue>107</blue>
      </color>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LightButton" name="widget3" native="true">
     <property name="bgColor" stdset="0">
      <color>
       <red>24</red>
       <green>189</green>
       <blue>155</blue>
      </color>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LightButton</class>
   <extends>QWidget</extends>
   <header>lightbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
